tests/lib/pkg_elf.c

```diff
@@ -58,27 +60,37 @@ ATF_TC_BODY(analyse_elf, tc)
 	ATF_REQUIRE(p != NULL);
 
 	ATF_REQUIRE_EQ(tll_length(p->shlibs_required), 0);
+	ATF_REQUIRE_EQ(pkg_analyse_elf(false, p, binpath, &provided, &provided_flags), EPKG_OK);
 	ATF_REQUIRE_EQ(tll_length(p->shlibs_provided), 0);
-	ATF_REQUIRE_EQ(pkg_analyse_elf(false, p, binpath), EPKG_OK);
-	ATF_REQUIRE_EQ(tll_length(p->shlibs_provided), 1);
-	ATF_REQUIRE_STREQ(tll_front(p->shlibs_provided), "libtestfbsd.so.1");
```

Comment:
This test stopped checking if libtestfbsd.so.1 is provided, which changes its behavior. Is this intended?